server/utils/utils.ts

```diff
@@ -21,3 +21,9 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
```

Comment:
I've taken the `asArray` function from your PR @leejacobson-moj cos it's ace and allows for a much neater style of coding in the controller 👍